content/en/docs/faq/troubleshooting.md

```diff
@@ -128,3 +128,45 @@ For example:
 helm list -v 6
 ```
 
+### Tiller installations stopped working and access is denied
+
+Helm releases used to be available from [https://storage.googleapis.com/kubernetes-helm/](https://storage.googleapis.com/kubernetes-helm/). As explained in ["Announcing get.helm.sh"](https://helm.sh/blog/get-helm-sh/), the official location changed in June 2019.
+
+The Tiller image [legacy location](https://gcr.io/kubernetes-helm/tiller) remained available until August 2021, and images began being removed at that time. [GitHub Container Registry](https://github.com/orgs/helm/packages/container/package/tiller) makes all the old Tiller images available.
```

Comment:
If this is a dead link, why link to it? :)

Reply:
There's some ambiguity because right now https://gcr.io/kubernetes-helm/tiller points to https://console.cloud.google.com/gcr/images/kubernetes-helm/GLOBAL/tiller?gcrImageListsize=30 and there are some images in there for 2.17 and 2.16.x. I'm trying to write a FAQ entry that will come up when people get confused and start searching to try to figure out what changed and when.

Reply:
Probably worth leaving in there as a reference for folks

Reply:
Edited to clarify